pkgs/tools/networking/ddclient/default.nix

```diff
@@ -1,24 +1,25 @@
-{buildPerlPackage, fetchurl, perlPackages, iproute}:
+{ stdenv, buildPerlPackage, fetchurl, perlPackages, iproute }:
 
-buildPerlPackage {
-  name = "ddclient-3.8.2";
+buildPerlPackage rec {
+  name = "ddclient-${version}";
+  version = "3.8.3";
 
   src = fetchurl {
-    url = mirror://sourceforge/ddclient/ddclient-3.8.2.tar.gz ;
-    sha256 = "17mcdqxcwa6c05m8xhxi4r37j4qvbp3wgbpvzqgmrmgwava5wcrw";
+    url = "mirror://sourceforge/ddclient/${name}.tar.gz";
+    sha256 = "1j8zdn7fy7i0bjk3jf0hxnbnshc2yf054vxq64imxdpfd7n5zgfy";
   };
 
   buildInputs = [ perlPackages.IOSocketSSL perlPackages.DigestSHA1 ];
 
-  patches = [ ./ddclient-foreground.patch ./ddclient-line-buffer-stdout.patch ];
```

Comment:
The `ddclient-foreground.patch` file? It is definitely gone inside my repo, [browsing the files](https://github.com/rycee/nixpkgs/tree/113106df1b8e2bbb1ba6cb8b11b36cd7be3e393d/pkgs/tools/networking/ddclient) at the commit does not list this file.

I think the diff view in GitHub is a bit misleading because it is a patch of a patch.